wiki/History_of_osu!/Mapping_and_Modding_Timeline/en.md

```diff
@@ -526,3 +526,75 @@ Mapping and modding systems are constantly improving. The **Mapping and modding
 - **2021-09-24:** [Quality Assurance checks](/wiki/People/The_Team/Beatmap_Nominators/General_Information#quality-assurance) no longer counted towards Beatmap Nominator activity. <!-- https://osu.ppy.sh/community/forums/topics/1423554?n=1 -->
   - This was implemented because it seemed like some users were using this as a loophole to avoid actively contributing as a Beatmap Nominator.
   - Prior to this change, every QA check was considered equal to 1/4 of a nomination.
+
+### October
+
+- **2021-10-12:** Former Beatmap Nominators who resigned on good terms within the past year can now become full members immediately upon rejoining.
+- **2021-10-13:** osu!mania beatmaps do not require hitsound additions anymore.
+  - This was done in response of a [popular request from the community](https://osu.ppy.sh/community/forums/topics/1418542) in order to allow for easier approachability to osu!mania mappers of different upbringings.
+- **2021-10-18:** Spread rules for osu!mania beatmaps have been relaxed.
+  - This was done upon request from the community via a [Ranking Criteria proposal](https://osu.ppy.sh/community/forums/topics/1440933) in order to make the under-utilized Ranked section more approachable to mappers.
+
+### December
+
+- **2021-12-04:** Following [this Ranking Criteria proposal](https://osu.ppy.sh/community/forums/topics/1447246), *Shokyuu* is now officially adopted as the official name of *Beginner* difficulties for osu!taiko.
+- **2021-12-14:** The burai sliders rule has been loosened per this [proposal](https://osu.ppy.sh/community/forums/topics/1442020).
+- This was done to accomodate for new mapping styles by making specific criteria for each difficuly, instead of making it a general rule.
```

Comment:
```suggestion
  - This was done to accomodate for new mapping styles by making specific criteria for each difficulty, instead of making it a general rule.
```